lib/bundles/inspec-compliance/api.rb

```diff
@@ -12,8 +12,8 @@ class API # rubocop:disable Metrics/ClassLength
     # logs into the server, retrieves a token and stores it locally
     def self.login(server, username, password, insecure)
       config = Compliance::Configuration.new
-      config['server'] = server
-      url = "#{server}/oauth/token"
+      config['server'] = "#{server}/api"
```

Comment:
I propose we add a separate `path` config, to cover cases where users proxy compliance behind another other e.g. http://myserver.com/compliance